server/config.ts

```diff
@@ -49,7 +54,9 @@ const auditConfig = () => {
 
 export default {
   buildNumber: get('BUILD_NUMBER', '1_0_0', requiredInProduction),
+  productId: get('PRODUCT_ID', 'UNASSIGNED', requiredInProduction),
```

Comment:
Do we also need this file from your other branch?
[helm_deploy/hmpps-education-and-work-plan-ui/values.yaml](https://github.com/ministryofjustice/hmpps-education-and-work-plan-ui/pull/539/files#diff-6792d52c6cda0e0a9247f7981e7e75dfdbf25ef09e52e294e9e58a28f5fc9cc4)